test/Serilog.Settings.Configuration.Tests/ConfigurationSettingsTests.cs

```diff
@@ -556,5 +559,43 @@ public void WriteToSubLoggerWithLevelSwitchIsSupported()
 
             Assert.Equal(1, DummyRollingFileSink.Emitted.Count);
         }
+
+        [Trait("Bugfix", "#103")]
+        [Fact]
+        public void MultipleArgumentValuesThrowsInvalidOperationException()
+        {
+            var jsonDiscreteValue = @"{
+                ""Serilog"": {            
+                    ""Using"": [""TestDummies""],
+                    ""WriteTo"": [{
+                        ""Name"": ""DummyRollingFile"",
```

Comment:
rolling file is deprecated, maybe example can start using `File` ?

Reply:
hm, I see its used in the rest of the test codebase here; prob leave it

Reply:
My thinking as well.